src/components/[guild]/AddRewardButton/useCreateTokenReward.ts

```diff
@@ -63,10 +70,18 @@ const useCreateReqBasedTokenReward = ({
 
   const addToExistingRole = async (data: CreateData, saveAs: "DRAFT" | "PUBLIC") => {
     if (data.roleIds.length > 1) {
+      captureEvent(
+        "Failed to add token reward (multiple roles selected)",
+        postHogOptions
+      )
       showErrorToast("Dynamic token rewards can be added to only one role at most.")
       return
     }
     if (data.requirements.length > 1) {
+      captureEvent(
+        "Failed to add token reward (multiple requirements set when adding to existing role)",
```

Comment:
In order to keep these event names short, we could just call them "Failed to add token reward" & pass the reason (e.g. "multiple requirements set when adding to existing role) in an event property ;)